demos-europe/demosplan-core
# fix(refs DPLAN-2153): remove file filter from assessment table filters in planfestsh

### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-2153/Abwagungstabellen-Filter-In-Planfest-ist-Datei-Bezug-nicht-moglich-entweder-Bezugs-Option-rein-oder-Filter-raus


This PR removes the file filter from the assessment table filters for the planfestsh project since it is not possible to create a file relation to a statement hence this filter is simply redundant.

### How to review/test
- checkout
- login as fpa user
- navigate to assessment table and select filters
- the file filter for statements should not be rendered anymore (only for the planfestsh project)

![grafik](https://github.com/user-attachments/assets/6fc08782-e4ff-4b06-9db2-36241eb4c376)

### PR Checklist


Delete the checkbox if it doesn't apply/isn't necessary.
